tests/phpunit/unit/tests/Shopping/Util.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests\Unit\Shopping;
+
+/**
+ * Class Util.
+ * Utility class to call / test non public methods of a Class
+ */
+class Util {
```

Comment:
Let's just copy this trait over and reuse it - https://github.com/GoogleForCreators/web-stories-wp/blob/main/tests/phpunit/integration/includes/Private_Access.php

Reply:
Okay updated 

https://github.com/GoogleForCreators/web-stories-wp/pull/11509/commits/3722d3d6874ee3fde8583a5d293135f89a034662